### DISCUSSION TITLE
Follow-Up and Action Points from Our Recent Meeting

### DISCUSSION THREAD

**Discussion by Rayccheal** [clentafrica/hr-people-operations#13] [General]
Hello Team Leads, @clentafrica/wg-blog @clentafrica/wg-design @clentafrica/wg-health @clentafrica/wg-education @clentafrica/wg-disability-gender-equality @clentafrica/wg-hr @clentafrica/wg-web-dev @clentafrica/wg-research-and-data @clentafrica/wg-partnerships @clentafrica/wg-education @clentafrica/wg-marketing-comms @clentafrica/wg-strategy @clentafrica/wg-sustainability-esg @clentafrica/wg-fundraising @clentafrica/wg-community-managers @clentafrica/wg-socialmedia 

I trust you are all doing well today!

I want to take a moment to appreciate all the Team Leads who attended our recent trailblazing meeting. Your insights, shared vision, and collective sense of responsibility are truly commendable, and I’m excited to be part of this dynamic team!

As we move forward, I encourage everyone to take action on the points we discussed to ensure maximum productivity in the coming weeks.

For those who could not attend, or if you need a refresher on the discussions, you can access the meeting notes [here](https://docs.google.com/document/d/1s8N5hq8TyYCmcc4EPkbjx4hqfBpVRFnU8wL3PVJGX7M/edit?usp=sharing). Additionally, if you are a Team Lead, and you haven’t yet joined the Team Leads group, you can do so [here](https://discord.gg/7WdtbCuM).

Thank you all, and let’s look forward to a highly productive week ahead!

Best regards,
Racheal

**Comment by emylajem** [comment]
Hello Good morning
I couldnt join the meeting because of bad network
I dont even know where i belong

**Reply by Rayccheal** [reply]
Hi @emylajem!

I hope you’re doing well!

Thank you for your feedback. It’s perfectly fine not to know where you belong yet! (lol) However, could you let us know which team you’re considering joining? I also recommend checking out the meeting notes as tagged above for more information on this and more.

I look forward to hearing from you and hopefully getting the issue of not knowing where you belong fixed as soon as we can.

Best,
Racheal

**Comment by Obutorbeatrice** [comment]
Same here

**Reply by Rayccheal** [reply]
Hi @Obutorbeatrice!

Here are a few things to help you stay up to date:

1. Which team are you considering joining?
2. Check out the meeting notes as tagged above, and we’ll follow up from there as soon as possible.
I look forward to hearing from you soon!

Best,
Racheal
